app/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiFhirImplementation.java

```diff
@@ -32,11 +31,13 @@ public <T extends IBase> Optional<T> fhirPathEvaluateFirst(
     @Override
     public <T extends IBaseResource> T parseResource(
             final String fhirResource, final Class<T> clazz) {
-        return PARSER.parseResource(clazz, fhirResource);
+        IParser resourceParser = CONTEXT.newJsonParser();
```

Comment:
I wasn't sure if the individual parsers needed individual names...?

Reply:
I think it's fine.